### ISSUE TITLE
Navigation fails on hosted site if "patterns" is in the path name

### ISSUE THREAD

**Issue by bshyong** [pattern-lab/patternlab-node#521]
I am using Pattern Lab Node `v2.5.1` on `Windows | Mac | Linux`, with Node `v5.9.0`, using the `Grunt` Edition.
##### Expected Behavior

Navigating in a pattern lab site deployed to a path containing the phrase `patterns` should not result in 404s.
##### Actual Behavior

I have a repo named vets.gov-patterns.
I've deployed a built version of our pattern lab to github pages, but Pattern Lab is manipulating the URLs behind the scenes which causes navigation to fail.

The website is hosted at https://department-of-veterans-affairs.github.io/vets.gov-patterns/

inside `function receiveIframeMessage(e)` is the following code:

```
                var i = /(patterns|snapshots)\/(.*)$/;
                t = window.location.protocol + "//" + window.location.host + window.location.pathname.replace(i, "") + a.path + "?" + Date.now(),
                window.location.replace(t)

```

basically our URLs are destroyed and the site attempts to navigate to vets.gov-NEWURL which results in a 404
##### Steps to Reproduce
- Deploy a built Pattern Lab site to any location with the phrase `patterns` in the path.
- Navigate the Pattern Lab site.

**Comment by geoffp** [comment]
I can confirm this. My team ran into this a while back, and we dodged the bug by changing our deployment directory, and I'd forgotten about it.

Thanks for the report, @bshyong.

**Comment by bmuenzenmeyer** [comment]
@dmolsen any ideas short of 
* changing the deployed site to not have `patterns`
* changing what PL is looking for? perhaps with a sg-patterns namespace ?
